library/ssl_msg.c

```diff
@@ -4866,6 +5001,15 @@ int mbedtls_ssl_parse_change_cipher_spec( mbedtls_ssl_context *ssl )
  *       and the caller has to make sure there's space for this.
  */
 
+static size_t ssl_transform_get_explicit_iv_len(
+                        mbedtls_ssl_transform const *transform )
```

Comment:
Style - Isn't the alignment out here?

Reply:
This, too, is because the line would be too long otherwise.